multigeiger/log_data.cpp

```diff
@@ -7,13 +7,12 @@
 int Serial_Print_Mode;
 
 static const char *Serial_Logging_Name = "Simple Multi-Geiger";
-static const char *dashes = "-----------------------------------------------------------------------------------------------------";
+static const char *dashes = "------------------------------------------------------------------------------------------------------------------------";
 
-static const char *Serial_Logging_Header = "     %10s %15s %10s %9s %9s %8s %9s %9s %9s";
-static const char *Serial_Logging_Body = "RAD: %10d %15d %10f %9f %9d %8d %9d %9f %9f";
+static const char *Serial_Logging_Header = "     %10s %15s %10s %9s %9s %8s %9s %9s %9s %5s %5s %6s";
+static const char *Serial_Logging_Body = "DATA %10d %15d %10f %9f %9d %8d %9d %9f %9f %5.1f %5.1f %6.0f";
```

Comment:
```
GEIGER: 1970-01-01T00:00:00      GMC_counts Time_difference Count_Rate Dose_Rate HV Pulses Accu_GMC Accu_Time Accu_Rate Accu_Dose  Temp  Humi  Press
GEIGER: 1970-01-01T00:00:00        [Counts]            [ms]      [cps]   [uSv/h]       [-] [Counts]      [ms]     [cps]   [uSv/h] [°C]   [%]  [hPa]
GEIGER: 1970-01-01T00:00:00 ------------------------------------------------------------------------------------------------------------------------
GEIGER: 1970-01-01T00:00:12 DATA         16           11300   1.415929  0.115311         6       16     11300  1.415929  0.115311  26.9  42.4  99501
```
The 2nd header line with the `[unit]` specs is misaligned.

Does anybody see **why**?

Reply:
@rexfue @t-pi ^^^ (both are using the Serial_Logging_Header format string)

Reply:
https://github.com/ecocurious2/MultiGeiger/pull/430/files#diff-ce17d7b8c1a55faffdf38cbd2357ed17c4ba28dc7067341bf869f1ba2d8e7fd2R28

Reply:
idea: replace degree char and percent char by harmless ascii char and compare.

Reply:
fixed: remove utf-8 encoded degree char disturbing formatted output
    
° (degree) char is encoded as 2 bytes, so C thinks it takes 2 char cells on the screen.
but it only renders as 1 char cell wide, so everything is rendered 1 position too far left.